src/sql/src/session/vars.rs

```diff
@@ -851,6 +851,18 @@ const STORAGE_DATAFLOW_MAX_INFLIGHT_BYTES: ServerVar<Option<usize>> = ServerVar
     internal: true,
 };
 
+///
```

Comment:
docs

and can we document that the above parameter is the on-off switch as well?

Reply:
yup, let me add that